README.md

```diff
@@ -1,57 +1,69 @@
-# gsw Python package
+# GSW-Python
 
-![https://travis-ci.org/TEOS-10/GSW-Python](https://travis-ci.org/TEOS-10/GSW-Python.svg?branch=master) ![https://conda.anaconda.org/conda-forge](https://anaconda.org/conda-forge/gsw/badges/installer/conda.svg) [![DOI](https://zenodo.org/badge/86503067.svg)](https://zenodo.org/badge/latestdoi/86503067)
+[![Tests](https://github.com/TEOS-10/GSW-Python/actions/workflows/tests.yml/badge.svg)](https://github.com/TEOS-10/GSW-Python/actions/workflows/tests.yml)
+[![Wheels](https://github.com/TEOS-10/GSW-Python/actions/workflows/cibuildwheel.yml/badge.svg)](https://github.com/TEOS-10/GSW-Python/actions/workflows/cibuildwheel.yml)
+[![DOI](https://zenodo.org/badge/86503067.svg)](https://zenodo.org/badge/latestdoi/86503067)
 
 
-This Python implementation of the Thermodynamic Equation of
-Seawater 2010 (TEOS-10) is based primarily on numpy ufunc wrappers of
-the GSW-C implementation.  We expect it to replace the original
-[python-gsw](https://github.com/TEOS-10/python-gsw)
-pure-python implementation after a brief overlap period.
-The primary reasons for this change are that by building on the
-C implementation we reduce code duplication and we gain an immediate
-update to the 75-term equation.  Additional benefits include a
-major increase in speed, a reduction in memory usage, and the
-inclusion of more functions.  The penalty is that a C (or MSVC C++ for
-Windows) compiler is required to build the package from source.
+This Python implementation of the Thermodynamic Equation of Seawater 2010 (TEOS-10) is based primarily on numpy ufunc wrappers of the GSW-C implementation.
+This replaces the original [python-gsw](https://github.com/TEOS-10/python-gsw) pure-python implementation after a brief overlap period.
+The primary reasons for this change are that by building on the C implementation we reduce code duplication and we gain an immediate update to the 75-term equation.  
+Additional benefits include a major increase in speed,
+a reduction in memory usage,
+and the inclusion of more functions.
+The penalty is that a C (or MSVC C++ for Windows) compiler is required to build the package from source.
 
-**Warning: this is for Python >=3.5 only.**
+**Warning: this is for Python >=3.6 only.**
```

Comment:
3.8?

Reply:
While I believe it can work on Python 3.6 you are right, we should mentioned only the versions that we are testing and packaging.